.env.example

```
@@ -1,3 +1,5 @@
 VITE_UPSTAGE_API_KEY=up_*********KwzE
 VITE_UPSTAGE_SYSTEM_PROMPT="You are a helpful assistant."
-VITE_API_BASE_URL=https://api.example.com
\ No newline at end of file
+VITE_API_BASE_URL=https://api.example.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_API_BASE_URL key should go before the VITE_UPSTAGE_API_KEY key

(UnorderedKey)